FE/src/components/Modal/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactNode } from 'react';
+import { CSSProp } from 'styled-components';
+
+import Potal from '@/components/Modal/Potal';
+
+import * as S from './style';
+
+export interface ModalProps {
+  wrapperId: string;
+  isOpen: boolean;
+  children: ReactNode;
+  onClose: () => void;
+  customStyle?: {
+    overlay: CSSProp | null | undefined;
+    content: CSSProp | null | undefined;
+    button: CSSProp | null | undefined;
+  };
+}
+
+export default function Modal({ wrapperId, isOpen, onClose, children, customStyle }: ModalProps) {
+  const { overlay, content, button } = S.defaultStyle;
+  const customOverlayStyle = customStyle?.overlay || overlay;
+  const customContentStyle = customStyle?.content || content;
+  const customButtonStyle = customStyle?.button || button;
```

Comment:
default parameter를 활용해보면 어떨까 싶습니다!
```suggestion
export default function Modal({ wrapperId, isOpen, onClose, children, customStyle = S.defaultStyle }: ModalProps) {
  const { overlay, content, button } = customStyle;
    ...
```